apps/web/pages/api/book/event.ts

```diff
@@ -373,6 +384,52 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
   // Initialize EventManager with credentials
   const rescheduleUid = reqBody.rescheduleUid;
+  type BookingType = {
```

Comment:
prisma.booking.findFirst below doesn't return any typed so I had to define this

Reply:
fixed for ya